packages/compass-connect/src/modules/connection-attempt.js

```diff
@@ -1,6 +1,8 @@
 import createDebug from 'debug';
+import createLogger from '@mongodb-js/compass-logging';
 import { promisify } from 'util';
 
+const { log, mongoLogId } = createLogger('COMPASS-CONNECT-UI');
```

Comment:
What about just `COMPASS-CONNECT` so we can make make a convention to name it 1-1 with the package name?

Reply:
Mhh … we can do that for now, I’ll switch it, but keep in mind that package names are implementation details and log component names are not, and they also might not map 1:1 semantically in some cases

Reply:
That's true, is a great point, we already know that we want to move stuff around! Would be nice to see if we can come up with some way / rule to get consistency .. for example the "UI" part catches my attention there .. should we always add "-UI" for all UI related logs? Not the end of the world if we don't do that though and we can just take a look later when we have more of the log calls and see if there are something that we want to align.

Reply:
Yeah, after thinking about it I’ve undone that switch, so, back to the beginning :)

I feel like adding `-UI` for UI components would be nice, yeah.